lib/v2/screens/app/interactor/usecases/get_incoming_deep_link.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:async/async.dart';
+// ignore: import_of_legacy_library_into_null_safe
+import 'package:seeds/features/scanner/telos_signing_manager.dart';
+import 'package:seeds/v2/datasource/local/settings_storage.dart';
+
+class GetIncomingDeepLinkUseCase {
```

Comment:
may be leave some comments explaning the difference between initial deep link and incoming deep link.